README.md

```diff
@@ -1,6 +1,9 @@
+# ⚠️This package is now deprecated⚠️
+It will receive no future updates. Please visit https://testcafe.io for instructions about installing and using the most recent TestCafe version.
```

Comment:
```suggestion
It will receive no future updates. Please migrate to the latest version of TestCafe. Learn about the latest version of TestCafe on https://testcafe.io. 
```